changelog.md

```diff
@@ -4,10 +4,37 @@ Since you are interested in what happens next, in case, you work for a for-profi
 
 ---
 
-### Bug fixes
+## 10.6.0 (2021-03-11)
 
 ### Improvements
 
+- [jss] `Styles` now supports `ClassNames`, `Props`/`Data`, and `Theme` as type parameters (eg. `Styles<Names, Data, Theme>`). [1460](https://github.com/cssinjs/jss/pull/1460)
+- [react-jss] `withStyles` and `createUseStyles` now support `ClassNames`, `Props`, and `Theme` as type parameters (eg. `createUseStyles<Names, Props, Theme>`). [1460](https://github.com/cssinjs/jss/pull/1460)
+- [react-jss] `useStyles` finally expects the correct argument type: a `Props` object with an optional `Theme` property (both determined from `createUseStyles`). [1460](https://github.com/cssinjs/jss/pull/1460)
+
+### Breaking Changes
+
+- [react-jss] `Theme` is no longer the first generic type parameter for `createUseStyles`. [1460](https://github.com/cssinjs/jss/pull/1460)
+  - There are two main ways to tell TypeScript your `Theme`'s type without reaching over the other type parameters:
+
+Using the function argument. (Recommended)
```

Comment:
I wouldn't encourage using this interface until we decide what to do with it, we had a discussion to remove it for various reasons

Reply:
Ah, I wasn't aware. I'll remove the "Recommended" piece then. Should the functional example be removed as well or should it stay?